tooling/api/scripts/after-build.cjs

```diff
@@ -17,7 +17,9 @@ if (!pkg.exports) {
 const outputPkg = {
   ...pkg,
   exports: Object.assign(
-    {},
+    {
+      './package.json': './package.json'
```

Comment:
why not just add this to `exports` in `package.json` instead of doing it in the `after-build.cjs`, it won't get overwritten.

Reply:
sounds weird either way LOL

Reply:
it is weird, I used to like `vite`, now I just hate all the bundlers.